var/spack/repos/builtin/packages/star/package.py

```diff
@@ -29,12 +29,11 @@ class Star(Package):
     """STAR is an ultrafast universal RNA-seq aligner."""
 
     homepage = "https://github.com/alexdobin/STAR"
-    url      = "https://github.com/alexdobin/STAR/archive/2.6.1a.tar.gz"
+    url      = "https://github.com/alexdobin/STAR/archive/2.6.1b.zip"
 
-    version('2.6.1a', '1ebaac553809c20900e0b42aadf75cc6')
+    version('2.6.1a', '87ae031376c904be1e0692f582beda7b')
     version('2.5.3a', 'baf8d1b62a50482cfa13acb7652dc391')
```

Comment:
@JusticeForMikeBrown if you change the package URL, it will substitute this version into that and download a zip, but this checksum is for the tar.gz, so there is an error:

```
==> Error: md5 checksum failed for /.../spack/var/spack/stage/star-2.5.3a-bfp4lwurbs2bjyprctbtxqecbcxrnywp/2.5.3a.zip
Expected baf8d1b62a50482cfa13acb7652dc391 but got cf8c2e64d715e73a45477a5b73cea8f8
```